### PR TITLE
Fixed #565 ddd layers and dot net 8

### LINKED ISSUE #565: DDD layers
**Describe the solution you'd like**
Create DDD's layers in project

### PR DESCRIPTION
**Pull request by sayyah**

### CODE REVIEW

**Review by pouryapb — APPROVED** [review]
Well, it runs...!

**Review by AliMozafari67 — DISMISSED** [review]

**Review by SamanR86 — DISMISSED** [review]

**Review by AliMozafari67 — DISMISSED** [review]

File: ECommerce.Application/ECommerce.Application.csproj [hunk 1]
```diff
@@ -6,8 +6,4 @@
     <Nullable>enable</Nullable>
```
Comment (pouryapb) [inline ECommerce.Application/ECommerce.Application.csproj:6]: should be net8.0. line 4

File: ECommerce.Front.BolouriGroup/appsettings.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
   },
   "FrontSetting": {
     "SiteName": "تجهیزات بلوری",
-    // "BaseAddress": "http://89.144.174.200:8080/"
-    "BaseAddress": "https://localhost:7257/"
+     "BaseAddress": "http://89.144.174.200:8080/"
+    //"BaseAddress": "https://localhost:7257/"
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/appsettings.json:11]: local default

File: EcommerceTestProject/ECommerce.ControllersTests.csproj [hunk 3]
```diff
@@ -27,7 +27,7 @@
```
Comment (pouryapb) [inline EcommerceTestProject/ECommerce.ControllersTests.csproj:27]: net8.0 here as well. line 4

**Review by pouryapb — CHANGES_REQUESTED** [review]

File: ECommerce.Front.BolouriGroup/Areas/Admin/Pages/Currencies/Index.cshtml [hunk 4]
```diff
@@ -55,31 +55,31 @@
             </form>
             <div class="example-preview table-responsive
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Areas/Admin/Pages/Currencies/Index.cshtml:56]: needs "> at the end of this line
```suggestion
            <div class="example-preview table-responsive">
```

File: ECommerce.Front.BolouriGroup/Areas/Admin/Pages/ProductComments/Accept.cshtml [hunk 5]
```diff
@@ -53,35 +53,35 @@
...
-                             <ul class="details-preview">                          
-                                @foreach (var image in Model.ProductComment.Product?.Images)
-                                {
-                                <img style="width: 25%" src="~/@($"{image.Path}/{image.Name}")" alt="@image.Alt">   
-                                }                               
-                            </ul>
-                        </div>                   
                     <div class="card-footer">
                         <button type="submit" class="btn btn-primary mr-2" asp-for="@Model.ProductComment.IsAccepted">تایید</button>
                         <a class="btn btn-secondary" asp-Page="/ProductComments/index" asp-area="Admin">لغو</a>
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Areas/Admin/Pages/ProductComments/Accept.cshtml:87]: at the end of this file missing this: >

**Review by pouryapb — CHANGES_REQUESTED** [review]

File: ECommerce.Front.BolouriGroup/Pages/Components/_compareProductCard.cshtml [hunk 6]
```diff
@@ -1,79 +1,79 @@
-﻿@model Ecommerce.Entities.ViewModel.ProductCompareViewModel
+﻿@model ProductCompareViewModel
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/_compareProductCard.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model ProductCompareViewModel
```

File: ECommerce.Front.BolouriGroup/Pages/Components/_productDetailsCart.cshtml [hunk 7]
```diff
@@ -1,35 +1,36 @@
-﻿@model Ecommerce.Entities.ViewModel.ProductIndexPageViewModel
+﻿@model ProductIndexPageViewModel
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/_productDetailsCart.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model ProductIndexPageViewModel
```

File: ECommerce.Front.BolouriGroup/Pages/Components/_productCard.cshtml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-﻿@model Ecommerce.Entities.ViewModel.ProductIndexPageViewModel
+﻿@model ProductIndexPageViewModel
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/_productCard.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model ProductIndexPageViewModel
```

File: ECommerce.Front.BolouriGroup/Pages/Components/_blogCard.cshtml [hunk 9]
```diff
@@ -1,4 +1,6 @@
-﻿@model Ecommerce.Entities.ViewModel.BlogViewModel
+﻿@using Microsoft.AspNetCore.Mvc.TagHelpers
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/_blogCard.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
```

File: ECommerce.Front.BolouriGroup/Pages/Shared/_Layout.cshtml [hunk 10]
```diff
@@ -1,105 +1,97 @@
-﻿@using Microsoft.Extensions.Configuration
-@inject IConfiguration Configuration;
+﻿@inject IConfiguration Configuration;
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Shared/_Layout.cshtml:1]:
```suggestion
@using Microsoft.Extensions.Configuration
@inject IConfiguration Configuration
```

File: ECommerce.Front.BolouriGroup/Pages/Components/CartList/Default.cshtml [hunk 11]
```diff
@@ -1,4 +1,4 @@
-@model IEnumerable<Ecommerce.Entities.ViewModel.PurchaseOrderViewModel>
+@model IEnumerable<PurchaseOrderViewModel>
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/CartList/Default.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model IEnumerable<PurchaseOrderViewModel>
```

**Review by pouryapb — CHANGES_REQUESTED** [review]

File: ECommerce.Front.BolouriGroup/Pages/Components/CartList/Default.cshtml (hunk 11 above)
Comment (pouryapb) [reply ECommerce.Front.BolouriGroup/Pages/Components/CartList/Default.cshtml:1]: also line 53, should change to `TempData["cartAllPrice"] = allPrice.ToString();` or there will be errors.

File: ECommerce.Front.BolouriGroup/Pages/Components/Username/Default.cshtml [hunk 12]
```diff
@@ -1,3 +1,3 @@
-﻿@model Ecommerce.Entities.ViewModel.LoginViewModel 
+﻿@model LoginViewModel
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/Username/Default.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model LoginViewModel 
```

File: ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMenu/Default.cshtml [hunk 13]
```diff
@@ -1,4 +1,4 @@
-﻿@model List<Ecommerce.Entities.ViewModel.CategoryParentViewModel>
+﻿@model List<CategoryParentViewModel>
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMenu/Default.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model List<CategoryParentViewModel>
```

File: ECommerce.Front.BolouriGroup/Pages/Components/BlogCategoriesMenu/Default.cshtml [hunk 14]
```diff
@@ -1,9 +1,9 @@
-﻿@model List<Ecommerce.Entities.ViewModel.CategoryParentViewModel>
+﻿@model List<CategoryParentViewModel>
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/BlogCategoriesMenu/Default.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model List<CategoryParentViewModel>
```

File: ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMobile/Default.cshtml [hunk 15]
```diff
@@ -1,4 +1,4 @@
-﻿@model List<Ecommerce.Entities.ViewModel.CategoryParentViewModel>
+﻿@model List<CategoryParentViewModel>
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMobile/Default.cshtml:1]:
```suggestion
@using ECommerce.Application.ViewModels
@model List<CategoryParentViewModel>
```

**Review by pouryapb — CHANGES_REQUESTED** [review]

File: ECommerce.Application/ECommerce.Application.csproj (hunk 1 above)
Comment (sayyah) [reply ECommerce.Application/ECommerce.Application.csproj:6]: This project is deleted.

File: EcommerceTestProject/ECommerce.ControllersTests.csproj (hunk 3 above)
Comment (sayyah) [reply EcommerceTestProject/ECommerce.ControllersTests.csproj:27]: This project is deleted.

File: ECommerce.Front.BolouriGroup/Areas/Admin/Pages/Currencies/Index.cshtml (hunk 4 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Areas/Admin/Pages/Currencies/Index.cshtml:56]: I fixed.

File: ECommerce.Front.BolouriGroup/Areas/Admin/Pages/ProductComments/Accept.cshtml (hunk 5 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Areas/Admin/Pages/ProductComments/Accept.cshtml:87]: I fixed.

File: ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMobile/Default.cshtml (hunk 15 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMobile/Default.cshtml:1]: I putted this namespace as a global namespace on project.

File: ECommerce.Front.BolouriGroup/Pages/Components/BlogCategoriesMenu/Default.cshtml (hunk 14 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Pages/Components/BlogCategoriesMenu/Default.cshtml:1]: I putted this namespace as a global namespace on project.

File: ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMenu/Default.cshtml (hunk 13 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Pages/Components/CategoriesMenu/Default.cshtml:1]: I putted this namespace as a global namespace on project.

File: ECommerce.Front.BolouriGroup/Pages/Components/Username/Default.cshtml (hunk 12 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Pages/Components/Username/Default.cshtml:1]: I putted this namespace as a global namespace on project.

Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Pages/Components/Username/Default.cshtml:1]: I putted this namespace as a global namespace on project.

File: ECommerce.Front.BolouriGroup/Pages/Components/CartList/Default.cshtml (hunk 11 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Pages/Components/CartList/Default.cshtml:1]: I putted this namespace as a global namespace on project.

I fixed.

**Review by AliMozafari67 — APPROVED** [review]

**Review by pouryapb — APPROVED** [review]

**Review by pouryapb — DISMISSED** [review]

**Review by pouryapb — APPROVED** [review]

**Review by AliMozafari67 — DISMISSED** [review]

File: ECommerce.Front.BolouriGroup/Areas/Admin/Pages/_ViewImports.cshtml [hunk 16]
```diff
@@ -1,4 +1,2 @@
 @namespace ECommerce.Front.BolouriGroup.Areas.Admin.Pages
```
Comment (pouryapb) [inline ECommerce.Front.BolouriGroup/Areas/Admin/Pages/_ViewImports.cshtml:1]: اینها رو هم پیدا کردم
```suggestion
@using ECommerce.Domain.Entities
@using ECommerce.Domain.Entities.Helper
@using ECommerce.Application.ViewModels
@namespace ECommerce.Front.BolouriGroup.Areas.Admin.Pages
```

**Review by pouryapb — CHANGES_REQUESTED** [review]

**Review by SamanR86 — APPROVED** [review]

File: ECommerce.Front.BolouriGroup/Areas/Admin/Pages/_ViewImports.cshtml (hunk 16 above)
Comment (sayyah) [reply ECommerce.Front.BolouriGroup/Areas/Admin/Pages/_ViewImports.cshtml:4]: I fixed.

**Review by pouryapb — DISMISSED** [review]

**Review by AliMozafari67 — APPROVED** [review]

**Review by Meysamhb — APPROVED** [review]

**Review by Moniy0o0 — APPROVED** [review]

**Review by Meysamhb — APPROVED** [review]

**Review by pouryapb — APPROVED** [review]

**Pull request merged**